cfme/tests/containers/test_static_custom_attributes.py

```diff
@@ -14,14 +21,43 @@
     pytest.mark.tier(2)]
 pytest_generate_tests = testgen.generate([ContainersProvider], scope='function')
 
+
+def get_random_string(length):
```

Comment:
After deep investigation, looks like fauxfactory cannot yield what I need, I need EXACTLY what get_random_string does and I think its functionality is not generic enough to add it to fauxfactory.
Let's keep this function here, it serves only the test scope.